include/gazebo_gps_plugin.h

```diff
@@ -102,7 +102,7 @@ class GAZEBO_VISIBLE GpsPlugin : public ModelPlugin
   static constexpr const double earth_radius = 6353000.0;      // meters
 
   // gps delay related
-  static constexpr double gps_update_interval_ = 0.2; // 5hz
+  double gps_update_interval_ = 0.2; // 5hz
```

Comment:
Any reason you would want to change the GPS rate? I mean the common rate for GPS is 5-10hz.

Reply:
to emulate gps receivers with rate more then 5hz